rules/Transform/Rector/Assign/PropertyFetchToMethodCallRector.php

```diff
@@ -199,7 +199,7 @@ private function matchPropertyFetchCandidate(PropertyFetch $propertyFetch): ?Pro
                 continue;
             }
 
-            return $propertyToMethodCall;
```

Comment:
This is wrong here. Rector is crashing here.

Reply:
Which rule is responsible?

Reply:
I fixed it on my side. Rector was maybe right here.